game-core/src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -861,9 +859,9 @@ private void doScrambling() {
         // Validate players have enough fuel to move there and back
         final Map<PlayerId, ResourceCollection> playerFuelCost = new HashMap<>();
         for (final Entry<Territory, Collection<Unit>> entry : toScramble.entrySet()) {
-          final Map<PlayerId, ResourceCollection> map =
+          final var map =
```

Comment:
Ditto with the variable name not being useful. The previous code relied more on the code knowing the type to make sense of what was happening. Perhaps renaming this here to `playerToResourcesMap` would make the type information unnecessary.